projects/addon-doc/src/components/navigation/navigation.component.ts

```diff
@@ -90,6 +101,7 @@ export class TuiDocNavigationComponent {
         this.open = false;
         this.menuOpen = false;
         this.search = '';
+        this.syncNavigationPanel();
```

Comment:
Maybe we can just move it to `title$.subscribe`?

Reply:
Not sure. It depends on how we expect it to work. If we move `this.syncNavigationPanel();` into `title$` subscription then we will have scrolling logic for manual navigation (user clicks on the link) too. If you think it's correct behavior I will move it there :) Here is how it works in this case:

https://user-images.githubusercontent.com/28801957/109471110-5a57b780-7a81-11eb-9ec6-67fb332202b6.mov

Reply:
We have `openPagesArr` and `openPagesGroupsArr`. Let's determine if we need to scroll or if they are already open?

Reply:
We need to scroll even when they are already opened, in the search and selection cases so I decided to split these into 2 parts – sync the side panel on init state and on search. I am using `title$` here just waiting for the end of the navigation